webpack.config.js

```diff
@@ -252,6 +287,71 @@ const GutenbergBlocksConfig = {
 	resolve: { alias },
 };
 
+// eslint-disable-next-line no-unused-vars
+const LegacyBlocksConfig = {
+	...GutenbergBlocksConfig,
+	output: {
+		path: path.resolve( __dirname, './build/' ),
+		filename: '[name]-legacy.js',
+		library: [ 'wc', 'blocks', '[name]' ],
+		libraryTarget: 'this',
+		// This fixes an issue with multiple webpack projects using chunking
+		// overwriting each other's chunk loader function.
+		// See https://webpack.js.org/configuration/output/#outputjsonpfunction
+		jsonpFunction: 'webpackWcBlocksJsonp',
+	},
+	plugins: [
```

Comment:
What do you think about moving the plugins configuration into a function `getBlocksPlugins( { fileSuffix: '-legacy' } )`, so we only define the configuration once instead of repeating it in `LegacyBlocksConfig` and `GutenbergBlocksConfig`?

Reply:
Ya, that half entered my mind while I was writing this but I thought it'd be nice to see if this is a direction we wanted to go first.  Since you noticed it too, I'll definitely do this.  However, on the whole, I think the entire webpack config could use some cleaning up (that can be done in its own pull though).

Reply:
> I think the entire webpack config could use some cleaning up (that can be done in its own pull though).

:100:

Reply:
Did a bit more than I intended but thought Why not? See af49e72